.teamcity/settings.kts

```diff
@@ -648,6 +652,33 @@ object SecurityAudit : BuildType({
     }
 })
 
+object QodanaAnalysis : BuildType({
+  name = "Qodana Analyze"
+
+  params {
+    password("env.QODANA_TOKEN", "******", display = ParameterDisplay.HIDDEN)
```

Comment:
I need help with access to the project on the Build Server to create an environment variable.

Reply:
Sure, I will contact in DM